src/main/resources/config/liquibase/changelog/20221120004207_changelog.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="pellumb" id="1668901346087-173">
```

Comment:
❌The id does not match out standards

Reply:
I changed the Id to our standards but now when i am trying to run again it gives the following error. How can i change the id without having this error?

Error: 

```Caused by: liquibase.exception.ValidationFailedException: Validation Failed:
     1 changesets check sum
          config/liquibase/changelog/20221120004207_changelog.xml::20221120004207::pellumb was: 8:d41d8cd98f00b204e9800998ecf8427e but is now: 8:75d3c1dd79f8608b2a60f7b7edefe3e3```

Reply:
You must reset the checksum on the db. There is a gradle command for it in our gradle file